gobblin-service/src/test/java/org/apache/gobblin/service/modules/orchestration/proc/ReevaluateDagProcTest.java

```diff
@@ -134,6 +146,12 @@ public void testOneNextJobToRun() throws Exception {
     // current job's state is deleted
     Assert.assertEquals(Mockito.mockingDetails(dagManagementStateStore).getInvocations().stream()
         .filter(a -> a.getMethod().getName().equals("deleteDagNodeState")).count(), 1);
+
+    Assert.assertEquals(Mockito.mockingDetails(this.dagActionReminderScheduler).getInvocations().stream()
+        .filter(a -> a.getMethod().getName().equals("unscheduleReminderJob")).count(), 1);
+
+    Assert.assertEquals(Mockito.mockingDetails(this.dagActionStore).getInvocations().stream()
+        .filter(a -> a.getMethod().getName().equals("deleteDagAction")).count(), 1);
```

Comment:
why are these called if the reminder fired and action was taken? I would expect deleteDagAction because it means the action has been completed (committed)

Reply:
this is testOneNextJobToRun. when there is no more job to run in re evaluate dag proc, it delete enforce_flow_finish_dag_action also